scripts/lib/upgrade-zulip-stage-2

```diff
@@ -22,14 +22,31 @@ os.environ["LANG"] = "en_US.UTF-8"
 os.environ["LANGUAGE"] = "en_US.UTF-8"
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
-from scripts.lib.zulip_tools import DEPLOYMENTS_DIR, su_to_zulip, assert_running_as_root
+from scripts.lib.zulip_tools import DEPLOYMENTS_DIR, su_to_zulip, \
+    assert_running_as_root, parse_os_release
 
 assert_running_as_root()
 
 logging.Formatter.converter = time.gmtime
 logging.basicConfig(format="%(asctime)s upgrade-zulip-stage-2: %(message)s",
                     level=logging.INFO)
 
+# Do not upgrade on unsupported OS versions.
+UNSUPPORTED_DISTROS = [
+    ('xenial', '16.04'),
```

Comment:
You mean `('ubuntu', '16.04')`.

Reply:
Yeah! It's bad that this commit hasn't been tested on a real machine.